doc/_themes/sphinx13/static/sphinx13.css

```diff
@@ -39,9 +39,8 @@ body {
     color: white;
     margin: 0;
     font-weight: 600;
```

Comment:
Optional, but maybe reduce the weight. Generally, the headings are sleek and a sleek project name looks more consistent and IMHO a bit more "designed§
```suggestion
    font-weight: 400;
```
before:
![image](https://github.com/sphinx-doc/sphinx/assets/2836374/a00e558a-aa87-46a2-b106-a28bc96031b1)

after:
![image](https://github.com/sphinx-doc/sphinx/assets/2836374/a4db13dd-e9af-4162-b94c-b764d3ebfd6e)

Reply:
done cheers